crates/iota-data-ingestion-core/src/executor.rs

```diff
@@ -23,17 +23,31 @@ use crate::{
 
 pub const MAX_CHECKPOINTS_IN_PROGRESS: usize = 10000;
 
+#[derive(Debug, Clone)]
+pub enum WorkerPoolMsg {
```

Comment:
Nit: I think this can be named more explicitly for clarity. This represents the status of the worker pool, which may be `running` (in which case we expect the progress info), or `shutdown` in which case we are free to handle it at will. So it sounds plausible to rename to
```rust
pub enum WorkerPoolStatus {
    Running((String, u64)), // This could by aliased by `type Progress`
    Shutdown,
}
```

Reply:
[fixed](https://github.com/iotaledger/iota/pull/4648/commits/f2d9f8da3328c5aae5815778d69daba4cdbd5bc9#diff-dfef1eeaea282ac0a41b60863611b2244808890824a073bf2264ca6fc775a624R30-R58)